internal/test/wmcb/wmcb_test.go

```diff
@@ -677,3 +677,42 @@ func (f *wmcbFramework) getLatestCniPluginsSHA() (string, error) {
 	sha512 := sha512AndFilename[0]
 	return sha512, nil
 }
+
+// initializeKubeNodePkgInfo populates the fileds of KubeNode pkg
+func (f *wmcbFramework) initializeKubeNodePkgInfo() error {
+	f.kubeNode = pkgInfo{}
+
+	f.kubeNode.url = kubeNodeBaseURL + framework.K8sVersion + "/kubernetes-node-windows-amd64.tar.gz"
+
+	kubeNodeSHA, err := f.getKubeNodeSHA()
+	if err != nil {
+		return fmt.Errorf("unable to get SHA for Kubernetes node binary for Windows: %v", err)
+	}
+	f.kubeNode.sha = kubeNodeSHA
+
+	f.kubeNode.shaType = "sha512"
+	return nil
+}
+
+// getKubeNodeSHA returns the SHA512 value of the kube node package.
+func (f *wmcbFramework) getKubeNodeSHA() (string, error) {
```

Comment:
Technically this could have been `kube package` struct method. No need of having it in framework right?